judge/jinja2/reference.py

```diff
@@ -18,8 +18,10 @@
 
 def get_user(username, data):
     if not data:
-        element = Element('span')
-        element.text = username
+        element = Element('del')
```

Comment:
Could we implement this as a CSS class rather a tag?

Reply:
Done. Created `.deleted-user`, and checked that the screenshot is still accurate.